### PR TITLE
Added bold feature to textArea.

### PR DESCRIPTION
**Pull request by jacmurphy**
*JIRA link*
https://tools.hmcts.net/jira/browse/ROC-2257

*Prototype*
https://hmcts-cmc-prototype.herokuapp.com/defendant/full/d-3/task-list/your-defence/partial

*Change description*

This PR allows textArea to be bold.
In this example "Any additional comments (optional)." is now bold.

### CODE REVIEW

File: macros/form.njk [hunk 1]
```diff
@@ -78,12 +78,13 @@
   {{ input(label, name, form, hint, 'email') }}
 {% endmacro %}
 
-{% macro textArea(label, name, form, inputClass='form-control-3-4', rows=5, labelClass='') %}
+{% macro textArea(label, name, form, inputClass='form-control-3-4', rows=5, labelClass='', bold = true) %}
   {% set error = form.errorFor(name) %}
   <div class="form-group {% if error %} form-group-error {% endif %}">
-    <label for="{{ name }}" id="{{ name }}[label]" class="form-label {{ labelClass }}">{{ t(label) }}</label>
+    <label for="{{ name }}" id="{{ name }}[label]" class="form-label{% if bold %}-bold{% endif %} {{ labelClass }}">
+      {{ label }}</label>
```
Comment (rlewan) [inline macros/form.njk]: Would you mind moving `</label>` to a new line?

Comment (rlewan) [reply macros/form.njk]: Also please translate the label.

File: macros/form.njk [hunk 2]
```diff
@@ -78,12 +78,13 @@
...
     {% if error %}
-      <span class="error-message">{{ t(error) }}</span>
+      <span class="error-message">{{ error }}</span>
```
Comment (rlewan) [inline macros/form.njk]: Please translate the error, our approach is that widgets should do the translations.

**Review by rlewan — CHANGES_REQUESTED** [review]
Please see my comments.

File: macros/form.njk (hunk 1 above)
Comment (jacmurphy) [reply macros/form.njk]: Updated

File: macros/form.njk (hunk 2 above)
Comment (jacmurphy) [reply macros/form.njk]: Updated

**Review by rlewan — APPROVED** [review]

File: macros/form.njk [hunk 3]
```diff
@@ -78,10 +78,12 @@
   {{ input(label, name, form, hint, 'email') }}
 {% endmacro %}
 
-{% macro textArea(label, name, form, inputClass='form-control-3-4', rows=5, labelClass='') %}
+{% macro textArea(label, name, form, inputClass='form-control-3-4', rows=5, labelClass='', bold = true) %}
```
Comment (timja) [inline macros/form.njk:81]: did you check if there's any places that shouldn't be bold? in the legal frontend as well?
The usual approach would be for non breaking changes, i.e. defaulting to false here

Comment (rlewan) [reply macros/form.njk:81]: That's our bad, let me fix that.

**Pull request merged**